tango/format.py

```diff
@@ -401,6 +402,29 @@ def __next__(self) -> str:
             raise StopIteration()
 
 
+@Format.register("sqlite_sequence")
+class SqliteSequenceFormat(Format[Sequence[T]]):
+    VERSION = 3
+
+    FILENAME = "data.sqlite"
```

Comment:
I feel like all `Format`s should make their file name or directory name transparent like this. Just a thought.

Reply:
I don't think that's always possible. Some formats, like the sqlite dataset dict format, need the directory structure. The sqlite dataset format stores the split names in filenames!

Reply:
Yes, same with `DatasetsFormat`, but in that case they could show the directory name. So we could call this `PATH` instead of `FILENAME`.

Reply:
There is no directory name though, from the perspective of the `Format` class. The format class is given a directory, and it should write into that directory. Sometimes it writes only one file, but sometimes it writes many.

Reply:
My point is that it can write it into a subdirectory of the given dir. This is how `DatasetsFormat` works.

Reply:
I don't know what all `Dataset.save_to_disk()` writes in there, but it should not create extra subdirectories if it doesn't have to. I think people will want to go into these directories and mess with the files themselves. This is, after all, the way we write out results. So those directories should be somewhat human readable.